EDEngineer/Resources/Data/localization.json

```diff
@@ -1952,144 +1952,6 @@
       "de": "Bestandteil in keinem bekannten Bauplan!",
       "fr": null,
       "ru": null
-    },

```

Comment:
copy paste :)
edit the file with a text editor then commit and push your change:

```
git commit -am"added back what I removed, and removed pesky F typo"
git push origin patch-1
```